CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+- More robust completion for referred keywords. If namespace is
```

Comment:
You might also mention your PR in the changelog entry, so it's easier to cross-references the changes.

Reply:
Done. Thanks!